tests/network-tests/src/Api.ts

```diff
@@ -661,9 +661,10 @@ export class Api {
   public async untilCouncilStage(
     targetStage: 'Announcing' | 'Voting' | 'Revealing' | 'Idle',
     announcementPeriodNr: number | null = null,
-    blocksReserve = 4,
+    blocksReserve = 1, // TODO dynamically adjust this as it stuck the election process
```

Comment:
What do you mean by "stuck the election process"?
I think there is a risk in having the reserve default to as low as `1`, as this may not be enough time for the integration tests to issue all transactions intended for a given stage (like announcing all candidacies during announcing stage) and we may experience some issues w/ the integration tests occasionally failing.

Reply:
> This is was due to the tests/network-tests/src/Api::untilCouncilStage final check for which the IdlePeriod (=1) >= reserve (=4) (in the playground config). So the playground election used to get stuck.
I have set reserve = 1 and then ran the local playground successfully

I will to set Idle period to 10 for the playground